artifacts/chart/templates/_karmada_webhook_configuration.tpl

```diff
@@ -0,0 +1,118 @@
+{{- define "karmada.webhook.configuration" -}}
+{{ $name :=  include "karmada.name" .}}
+{{ $namespace := include "karmada.namespace" .}}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-config
```

Comment:
should we provide a option if user uses cert-manager for CA injection ?

Reply:
More options will be provided in the future.